src/execution_plan/ops/op_expand_into.c

```diff
@@ -97,30 +69,27 @@ OpBase *NewExpandIntoOp(const ExecutionPlan *plan, Graph *g, AlgebraicExpression
 	op->graph = g;
 	op->ae = ae;
 	op->r = NULL;
-	op->edges = NULL;
 	op->F = GrB_NULL;
 	op->M = GrB_NULL;
 	op->recordCount = 0;
-	op->edgeRelationTypes = NULL;
-	op->recordsCap = 0;
 	op->records = NULL;
+	op->recordsCap = 0;
```

Comment:
It feels a waste to use `rm_calloc` but then set explicit NULLs,
if we still want to do this, we should set `edge_ctx` to NULL at this point.